pkg/kubelet/cm/cpuset/cpuset.go

```diff
@@ -319,6 +319,9 @@ func Parse(s string) (CPUSet, error) {
 			if err != nil {
 				return NewCPUSet(), err
 			}
+			if start >= end {
```

Comment:
The cpuset `1-1` should be accepted as `1`, or?

```suggestion
			if start > end {
```

Reply:
In my opinion asking for a range of "1-1" seems likely to be a typo, since "1" would be the most likely way of expressing that intent. So I'd err on the side of rejecting it.

Reply:
Just added an explicit test case for this "1-1" case.

Reply:
Still not sure if I agree that `1-1` or `10-10` is a typo. It has been supported previously, and I think we should have a good reason to deny such a range.

Simple scripts like `echo 0-$(($(nproc)-1))` will return `0-0` on single core machines eg.